### PR TITLE
refactor(ui5-list): removed ui5-item-press event

### PR DESCRIPTION
**Pull request by nnaydenow**
Removed usage of undocumented ui5-item-press event. In addition controls which were using are refactored to use ui5-selection-change event of the ui5-list. With #4448 ui5-item-click event is fired before ui5-selection-change in order to cancel the click / selection if needed and the event doesn't provide information if the clicked item is selected.

### CODE REVIEW

**Comment by nnaydenow** [comment]
Hello @SAP/ui5-webcomponents-topic-rd and @SAP/ui5-webcomponents-topic-rl,

Could you please check if everything works okey with the components that you are responsible for?

Regards,
Nayden

File: packages/main/src/TabContainer.js [hunk 1]
```diff
@@ -398,7 +398,7 @@ class TabContainer extends UI5Element {
 	}
 
 	_onOverflowListItemSelect(event) {
-		this._onItemSelect(event.detail.item);
+		this._onItemSelect(event.detail.selectedItems[0]);
```
Comment (gmkv) [inline packages/main/src/TabContainer.js:401]: save this to a variable and reuse it on line 403 as well. otherwise it throws

**Review by gmkv — CHANGES_REQUESTED** [review]

**Review by gmkv — APPROVED** [review]

**Comment by gmkv** [comment]
LGTM from @SAP/ui5-webcomponents-topic-rd

**Review by kineticjs — APPROVED** [review]

File: packages/main/src/TabContainerPopover.hbs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 	_hide-header
 	class="ui5-tab-container-responsive-popover"
 >
-	<ui5-list separators="None" @ui5-item-press="{{_onOverflowListItemSelect}}">
+	<ui5-list mode="SingleSelect" separators="None" @ui5-selection-change="{{_onOverflowListItemSelect}}"></ui5-list>
```
Comment (gmkv) [inline packages/main/src/TabContainerPopover.hbs:11]: do not place a closing tag here

**Review by gmkv — CHANGES_REQUESTED** [review]

File: packages/main/src/TabContainerPopover.hbs (hunk 2 above)
Comment (nnaydenow) [reply packages/main/src/TabContainerPopover.hbs:11]: resolving conflicts mistake. thanks!

**Comment by MapTo0** [comment]
lgtm

**Review by MapTo0 — APPROVED** [review]

**Review by gmkv — APPROVED** [review]

**Review by niyap — APPROVED** [review]

**Review by plamenivanov91 — APPROVED** [review]

**Pull request merged**